### PR TITLE
[UI] Added app icon

### PR DESCRIPTION
**Pull request by NicolasR27**
just added the app icon png from resource  folders.

### CODE REVIEW

**Comment by NicolasR27** [comment]
added missing icon  files  complete now

File: Say Their Names.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1090,7 +1090,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = Y9UNY8645A;
```
Comment (hybridcattt) [inline Say Their Names.xcodeproj/project.pbxproj:1093]: could you revert changes to the dev team? 🙏  then we are ready to merge!

File: Say Their Names.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1111,7 +1111,7 @@
...
```
Comment (hybridcattt) [inline Say Their Names.xcodeproj/project.pbxproj:1114]:
```suggestion
				DEVELOPMENT_TEAM = "";
```

**Comment by evilpenguin** [comment]
@NicolasR27 Can you please fix the conflicts?

**Comment by evilpenguin** [comment]
Closing. This work has been done in #105

**Pull request closed without merging**